tests/testthat/helpers-with-settings.R

```diff
@@ -24,10 +24,22 @@ app_tws <- function() {
           rtables::analyze(c("SEX", "AGE"))
 
         tbl <- rtables::build_table(l, df1)
-
         tbl
       })
       table_with_settings_srv(id = "table_with_settings", table_r = table_r)
     }
   )
 }
+
```

Comment:
maybe rename file to helpers.R, cause we might add other functions here, used by `table-with-settings` and other

Reply:
I understand they can have own helper-NAME files.